docs/resources/elasticsearch.md.erb

```diff
@@ -134,7 +116,7 @@ Returns a list of enabled modules for each node in the cluster. For more additio
 
 ### modules
 
-Returns detailed information about each enabled module for each node in the cluster. For a succint list of the names of each of the modules enabled, use the `module_list` property. This example uses a bit of additional Ruby to find a specific module and assert a value.
+Returns detailed information about each enabled module for each node in the cluster. For a succint list of the names of each of the modules enabled, use the `module_list` property. This example uses a additional Ruby to find a specific module and assert a value.
```

Comment:
Can we rephrase "This example uses a additional Ruby" to just "This example uses additional Ruby". This is what we have later on and I think it sounds better.